addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -3014,6 +3014,12 @@ const SnippetOptionWidget = Widget.extend({
      * @type {boolean}
      */
     forceNoDeleteButton: false,
+    /**
+     * Forces the target to not be possible to clone.
+     *
+     * @type {boolean}
+     */
+    forceNoCloneButton: false,
```

Comment:
Should we keep only one attribute to hide both buttons? I don't think it ever make sense to have the possibility to duplicate but not to remove? (otherwise it means that if you duplicate, you cannot remove the thing you just duplicated?).

Strange we did not think about it when implementing forceNoDeleteButton TBH :shrug:

Reply:
As you prefer, I tried to keep things "clear" by their name. So you prefer if I make `forceNoDeleteButton` remove also the duplicate button behind the scene regardless of the name of the option? And in master, find a better name?

Reply:
Yep, I would do that :+1:

Reply:
Adapted https://github.com/odoo/odoo/compare/9b3f277c9080dbdd4c74a478a3990a0278b6c63e..cff65ed64bb85cbf6eeb29951e979c0bcf69017b

Maybe we don't even need to change the option name in master, it's just implicit that hiding the delete hides the duplicate. There might be cases where you want to have one and not the other tho (like being able to delete but not duplicate), could be separate option one day